UPGRADING.md

```diff
@@ -0,0 +1,19 @@
+## Upgrading Mongoid Locker
+
+### Upgrading to 1.0.0
+Locker is now doesn't define `:locked_at` and `:locked_until` fields by default during include.
+
+By default, Locker still uses fields with `:locked_at` and `:locked_until` names which should be defined manually in a model.
+```ruby
+class User
+  include Mongoid::Document
+  include Mongoid::Locker
+
+  field :locked_at, type: Time
+  field :locked_until, type: Time
+end
+```
+
+Field names used by Locker can be defined with `locker` class method or through global `configure`. See [Customizable :locked_at and :locked_until field names](https://github.com/mongoid/mongoid-locker#customizable-locked_at-and-locked_until-field-names) for more information.
```

Comment:
You can customize the fields used with a `locker` class method or via a global  `configure`.